src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,136 @@
+            while (resultSet.next()) {
+                String manufacturerName = resultSet.getString("name");
+                String manufacturerCountry = resultSet.getString("country");
+                Long id = resultSet.getObject("id", Long.class);
+                Manufacturer manufacturer = new Manufacturer();
+                fillManufacturer(manufacturer, manufacturerName, manufacturerCountry, id);
+                allManufacturers.add(manufacturer);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all manufacturers from db", e);
+        }
+        logger.info("The all data from DB was successful fetched");
+        return allManufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        logger.info("Method update was called with manufacturer: " + manufacturer);
+        String updateManufacturerRequest =
+                "UPDATE manufacturers SET name = ?, country = ? WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(updateManufacturerRequest)) {
+            createManufacturerStatement.setString(1, manufacturer.getName());
+            createManufacturerStatement.setString(2, manufacturer.getCountry());
+            createManufacturerStatement.setObject(3, manufacturer.getId());
+            createManufacturerStatement.executeUpdate();
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can`t update manufacturer from db "
+                    + manufacturer, throwables);
+        }
+        logger.info("Update data of manufacturer was successful");
+        return manufacturer;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        logger.info("Method delete was called with id: " + id);
+        String deleteManufacturerRequest =
+                "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(deleteManufacturerRequest)) {
+            createManufacturerStatement.setLong(1, id);
+            return createManufacturerStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t delete manufacturer from db by id " + id, e);
+        }
+    }
+
+    private Manufacturer fillManufacturer(Manufacturer manufacturer,
+                                          String manufacturerName,
+                                          String manufacturerCountry,
+                                          Long id) {
+        manufacturer.setName(manufacturerName);
+        manufacturer.setCountry(manufacturerCountry);
+        manufacturer.setId(id);
+        return manufacturer;
```

Comment:
not like this, pass a result set as a parameter and create manufacturer right here
```suggestion
    private Manufacturer parseManufacturer(ResultSet resultSet) {
        // new manufacturer
        // set its fields
        // return this new manufacturer
```